src/helpers/find-titles.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+const findTitles = (titles) => {
+  const h1List = [];
+  const h2List = [];
+  const h3List = [];
+  titles.map((title) => {
+    if (title.nodeName === "H1") {
+      if (title.innerHTML.includes('<strong>')) {
+        title.innerHTML = title.innerHTML.slice(8);
+        title.setAttribute("id", title.innerHTML);
+        h1List.push(title);
+      } else {
+        title.setAttribute("id", title.innerHTML);
+        h1List.push(title);
+      }
```

Comment:
Avoid duplicate code:
```suggestion
      if (title.innerHTML.includes('<strong>')) {
        title.innerHTML = title.innerHTML.slice(8);
      } 
     title.setAttribute("id", title.innerHTML);
     h1List.push(title);
```

Reply:
Done